src/internal/model/model.go

```diff
@@ -53,6 +53,16 @@ const (
 	// manifest search. It shouldn't be used to differentiate between backups once
 	// the manifest search completes.
 	MergeBackup = "merge-backup"
+	// PreviewBackup denotes that this backup contains a subset of information for
+	// the protected resource. PreviewBackups are used to demonstrate value but
+	// are not safe to use as merge bases for incremental backups. It's possible
+	// they could be used as assist bases since the only difference from a regular
+	// backup is the amount of data they contain.
+	//
+	// This tag should only be used for finding backups during a
+	// manifest search. It shouldn't be used to differentiate between backups once
```

Comment:
> It shouldn't be used to differentiate between backups once the manifest search completes.
Could you explain this a bit?

Reply:
sure! I'll update the comment in the code as well to make it more clear

Mostly I'm trying to say that it's alright to use these tags during `FindBases()` but we shouldn't use them after that. Outside of `FindBases()` we should really use something like the `BackupBases` struct's functions or something to differentiate how a previous backup will be used in the current backup